ttnn/cpp/ttnn/operations/matmul/device/matmul_op_multi_core_reuse_optimized_program_factory.cpp

```diff
@@ -206,6 +206,8 @@ operation::ProgramWithCallbacks create_program(
         in1_per_core_w,       // in1_per_core_w
 
         num_blocks,  // num_blocks
+        1,
```

Comment:
could you please add comment to represent that it's output blocks number (information to make sure that it's clear what this one represents)?

Reply:
done